app/views/hosts/_selected_hosts.html.erb

```diff
@@ -1,4 +1,6 @@
 <div class="row">
+  <%= alert(:class => 'alert-info hide', :close => true, :header => '',
```

Comment:
I'm looking for a way to say that you are seeing only the first 20 but you actually selected all of them, so if you take an action against them its the entire list not just he one you see (same workflow if you select all of your emails on gmail).